java/src/IceGridGUI/build.gradle

```diff
@@ -3,32 +3,13 @@
 project.ext.displayName = "IceGridGUI"
 project.ext.description = ""
 
-ext.hasJavaFx = false
-
-if(iceGridGuiUseJavaFX.toBoolean()) {
-    ext.os = System.properties['os.name']
-    if(os == "Mac OS X") {
-        ext.platform = "mac"
-        ext.hasJavaFx = true
-    } else if(os.toLowerCase().contains("windows")) {
-        ext.platform = "win"
-        ext.hasJavaFx = true
-    } else if(os.toLowerCase().contains("linux")) {
-        ext.platform = "linux"
-        ext.hasJavaFx = true
-    } else {
-        ext.hasJavaFx = false
-    }
```

Comment:
Should we avoid setting `ext.os` for an unknown platform like we used to? Instead of defaulting to `linux` now